book/07-git-tools/sections/revision-selection.asc

```diff
@@ -171,9 +169,15 @@ Date:   Thu Dec 11 15:08:43 2008 -0800
     Merge commit 'phedders/rdocs'
 ----
 
-It’s important to note that the reflog information is strictly local – it’s a log of what you’ve done in your repository.
-The references won’t be the same on someone else’s copy of the repository; and right after you initially clone a repository, you'll have an empty reflog, as no activity has occurred yet in your repository.
-Running `git show HEAD@{2.months.ago}` will work only if you cloned the project at least two months ago – if you cloned it five minutes ago, you’ll get no results.
+It’s important to note that reflog information is strictly local -- it’s a log only of what _you've_ done in _your_ repository.
+The references won’t be the same on someone else’s copy of the repository; also, right after you initially clone a repository, you'll have an empty reflog, as no activity has occurred yet in your repository.
+Running `git show HEAD@{2.months.ago}` will work only if you cloned the project at least two months ago -- if you cloned it five minutes ago, you’ll get no results.
```

Comment:
I think what this does is find the commit _closest_ to the date/time you specify. So if I have a two-weeks-old project and I do `git show HEAD@{2.years.ago}` I'll get the initial commit.

Reply:
Let me verify that behaviour.

Reply:
Yup, that's what it appears to do. You do know that that was *your* sentence, right? :-)

Reply:
Yup. 😊